### DISCUSSION TITLE
快捷指令折腾之路 | Goalonez

### DISCUSSION THREAD

**Discussion by Goalonez** [Goalonez/goalonez.github.io#165] [Announcements]
https://blog.goalonez.site/blog/%E5%BF%AB%E6%8D%B7%E6%8C%87%E4%BB%A4%E6%8A%98%E8%85%BE%E4%B9%8B%E8%B7%AF.html 

 Goalonez Blog

**Comment by Icedroid** [comment]
直立 倒立动作没有呀，报错了

**Comment by Goalonez** [comment]
> @Icedroid
> 直立 倒立动作没有呀，报错了
> 

我这边测试是正常的。发布blog至今并没有改过指令内容，正常来说应该不会有问题。是不是没下actions的app。

**Comment by Icedroid** [comment]
> @Goalonez
> > @Icedroid
> > 直立 倒立动作没有呀，报错了
> > 
> 
> 我这边测试是正常的。发布blog至今并没有改过指令内容，正常来说应该不会有问题。是不是没下actions的app。

哦，是了，谢谢，没有注意要下载软件。
